samples/java/README.MD

```diff
@@ -32,24 +32,66 @@ $ mvn install:install-file -Dfile=target/azure-functions-java-library-kafka-1.0.
 
 ## Build the sample 
 
-Go to the sample app directory, buid the sample.
+### Go to sample directory
 
 ```
 $ cd ../../sample
-$ mvn clean
-$ mvn package
 ```
 
-## Install the KafkaTriggerExtension 
+### Modify pom.xml (optional)
+
+Go to the sample app directory, buid the sample. If you want to change the function App that you want to deploy this app, change the following section on the `pom.xml` file. The current `pom.xml` file is for windows and deploy windows based function app. If you are using mac, you can rename the `pom_linux.xml` to `pom.xml`. It deploys linux based function app.
+
+_pom.xml_  
+
+```xml
+        <functionAppName>kafka-function-20190419163130420</functionAppName>
+        <functionAppRegion>westus</functionAppRegion>
+        <stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>
+        <functionResourceGroup>java-functions-group</functionResourceGroup>
+```
+
+### Modify install_extension shell
+
+Modify the functionAppName of `install_extension.ps1` (windows) or `install_extension.sh` (mac/bash) if you change the functionAppName. 
+
+_isntall_extension.ps1_
+
+```powershell
+$FunctionAppName = "kafka-function-20190419163130420"
+```
+
+_install_extension.sh_
+
+```bash
+FUNCTION_APP_NAME=kafka-function-20190419163130420
+```
 
-Go to the Azure Functions directory that is created by the package command. Please change the number following `kafka-function-` according to your environment. 
+### Modify TriggerFunction.java (Mac/Linux user only)
+
+Remove following line. This line is windows only. 
+
+```java                 
+sslCaLocation = "confluent_cloud_cacert.pem", // Remove this attribute for Linux/Mac users.
```

Comment:
Why is only for linux/mac, not windows?

Reply:
Linux and mac works without this. Windows requires. Probably, the default cacert already includes it. (I didn't need it when I use Mac and deploy to Linux Premium)

Reply:
The original documentation mention it. https://github.com/confluentinc/examples/tree/5.4.0-post/clients/cloud/csharp#produce-records